eu.octanne.McBoyard.yml

```diff
@@ -0,0 +1,64 @@
+app-id: eu.octanne.McBoyard
+separate-locales: false
+
+base: io.atom.electron.BaseApp
+base-version: '19.08'
+
+runtime: org.freedesktop.Platform
+runtime-version: '19.08'
+
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+
+finish-args:
+  - --socket=fallback-x11
+  - --share=ipc
+  - --share=network
+  - --device=dri
+  - --socket=pulseaudio
+
+  - --filesystem=home
```

Comment:
What does it need home access for?

Reply:
It needs home access to add a ".mcboyard" folder

Reply:
You can use `--presist=.mcboyard` the application doesn't provide a way to override it so that we can make it use the sandboxed xdg-data dir for config/cache?

Reply:
I just tested and it works with `--persist=.McBoyard`, thanks!